cmake/external/GoogleTestNMock.cmake

```diff
@@ -8,8 +8,8 @@ SET (GMOCK_REVISION 443)
 include(ExternalProject)
 ExternalProject_Add(
 	GoogleMock
-	URL http://googlemock.googlecode.com/files/gmock-1.7.0.zip
-	URL_MD5 073b984d8798ea1594f5e44d85b20d66
+	URL https://github.com/google/googletest/archive/release-1.8.0.tar.gz
```

Comment:
please cherry pick the google mock, separate patch for that, before all the others